csvcubed/csvcubed/schema/codelist-config/v1_0/schema.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "id": "https://purl.org/csv-cubed/code-list-config/v1.0",
+  "title": "Schema for a codelist.",
+  "description": "Schema to be used for creation and validation of a single codelist in CSV-W format.",
+  "type": "object",
+  "required": [],
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "default": "https://purl.org/csv-cubed/code-list-config/v1.0"
+    },
+    "id": {
+      "description": "A unique id for the code list.",
+      "type": "string"
+    },
+    "title": {
+      "description": "A title for the code list.",
+      "type": "string"
+    },
+    "description": {
+      "description": "A long description of the code list.",
+      "type": "string"
+    },
+    "summary": {
+      "description": "A short summary of the code list.",
+      "type": "string"
+    },
+    "creator": {
+      "description": "The creator of the code list (uri).",
+      "$ref": "http://purl.org/csv-cubed/resources/organisations.json#/uris"
+    },
+    "publisher": {
+      "description": "The publisher of the code list (uri).",
+      "$ref": "http://purl.org/csv-cubed/resources/organisations.json#/uris"
+    },
+    "dataset_issued": {
+      "description": "Date time that the code list was initially published in ISO 8601 format, e.g. 2022-03-31 or 2022-03-31T12:54:30Z.",
+      "type": "string"
+    },
+    "dataset_modified": {
+      "description": "Date time that the code list was last modified published in ISO 8601 format, e.g. 2022-03-31 or 2022-03-31T12:54:30Z.",
+      "type": "string"
+    },
+    "license": {
+      "description": "URI that represents the copyright license applying to this code list.",
+      "$ref": "http://purl.org/csv-cubed/resources/licenses.json#/uris"
+    },
+    "themes": {
+      "$ref": "#/definitions/v1.0/themeTypes"
+    },
+    "keywords": {
+      "$ref": "#/definitions/v1.0/keywordTypes"
+    },
+    "sort": {
+      "description": "Sort strategy for the concepts.",
+      "type": "object",
+      "required": [
+        "by",
+        "method"
+      ],
+      "properties": {
+        "by": {
+          "description": "labels (default) or notation",
```

Comment:
It'd be better to create enums here, so users will be given the auto-complete options. The same is true for the  `method` property too.